pkg/networkservice/common/tracehelper/context.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 Cisco Systems, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracehelper
+
+import (
+	"context"
+)
+
+const (
+	connectionInfoKey contextKeyType = "ConnectionInfo"
+)
+
+type contextKeyType string
+
+// ConnectionInfo - struct, containing string representations of request and response, used for tracing.
+type ConnectionInfo struct {
+	// Request is string representation of networkservice.NetworkServiceRequest
+	Request string
```

Comment:
I would prefer to see a protobuf message here.

Reply:
Replaced with `proto.Message`